lib/index.js

```diff
@@ -848,7 +848,7 @@ var Client = module.exports = function (config) {
           callCallback(err)
         } else {
           headers['content-length'] = stat.size
-          headers['content-type'] = mime.lookup(msg.name)
+          headers['content-type'] = mime.getType(msg.name)
```

Comment:
see https://www.npmjs.com/package/mime#version-2-notes, sorry that I missed that